package.json

```diff
@@ -69,7 +70,8 @@
     "redux-thunk": "^2.4.1",
     "reselect": "^4.1.5",
     "tinymce": "^5.10.2",
-    "lodash-es": "^4.17.21"
+    "video-react": "^0.15.0",
+    "video.js": "^7.18.1"
```

Comment:
Pre sync questions: makes sense we are adding both now, before we become stuck to one, but we should run through the pros and cons of each.